jsonargparse_tests/test_parameter_resolvers.py

```diff
@@ -899,6 +902,13 @@ def test_get_params_some_ignored():
         assert_params(get_params(func_given_kwargs), ["p", "p1"], help=False)
 
 
+def test_partialmethod():
+    ClassA.partial_method_a = partialmethod(ClassA.method_a, pma1=1, pma2=0.5)
+    assert_params(get_params(ClassA, "partial_method_a"), ["pma1", "pma2", "kma1"])
```

Comment:
Looking again at this test now I think there are things missing. The partial doesn't change the list of parameter names. But it does change the defaults. Are the defaults correctly inferred by a parser? If not, then it doesn't really work as expected. So this should be tested.

And another thing that could be considered missing is that a method could be set as default to a callable. For this `--print_config` should serialize this default into an import path string. And this import path when used in a config, should be correctly deserialize into the partial method object.